### ISSUE TITLE
Implement Comprehensive Error Handling in Linear Regression Window

### ISSUE THREAD

**Issue by Jawad-Farrukh-665** [Jawad-Farrukh-665/ML-Studio#14]
### **Feature Request:**
 I'd like to propose the addition of robust error handling in linear regression window to ensure smoother user experience and better debugging. Specifically, we should handle:

1. If an invalid dataset is uploaded i.e. the dataset with `.docx` or `.pptx`, etc. extension.
2. For now only the `.csv` extension is acceptable.
3. Gradient Descent option is selected but no learning rate or number of iterations is entered.
4. No Target or Predictor Variable is selected
5. No Linear Regression type is selected
